adapters/AppLovin/AppLovinAdapter/GADMediationAdapterAppLovin.m

```diff
@@ -20,53 +20,65 @@
 #import "GADMRTBAdapterAppLovinBannerRenderer.h"
 #import "GADMRTBAdapterAppLovinInterstitialRenderer.h"
 
-@interface GADMediationAdapterAppLovin ()
-@property(nonatomic, strong) GADMAdapterAppLovinRewardedRenderer *rewardedRenderer;
-@property(nonatomic, strong) GADMRTBAdapterAppLovinBannerRenderer *bannerRenderer;
-@property(nonatomic, strong) GADMRTBAdapterAppLovinInterstitialRenderer *interstitialRenderer;
-@end
+@implementation GADMediationAdapterAppLovin {
+  /// AppLovin banner ad wrapper.
+  GADMRTBAdapterAppLovinBannerRenderer *_bannerRenderer;
+
+  /// AppLovin interstitial ad wrapper.
+  GADMRTBAdapterAppLovinInterstitialRenderer *_interstitialRenderer;
 
-@implementation GADMediationAdapterAppLovin
+  /// AppLovin rewarded ad wrapper.
+  GADMAdapterAppLovinRewardedRenderer *_rewardedRenderer;
+}
 
-+ (void)setUpWithConfiguration:(GADMediationServerConfiguration *)configuration
-             completionHandler:(GADMediationAdapterSetUpCompletionBlock)completionHandler {
-  // Compile all the SDK keys we should initialize for
++ (void)setUpWithConfiguration:(nonnull GADMediationServerConfiguration *)configuration
+             completionHandler:(nonnull GADMediationAdapterSetUpCompletionBlock)completionHandler {
+  // Compile all the SDK keys that should be initialized.
   NSMutableSet<NSString *> *sdkKeys = [NSMutableSet set];
   NSObject *sdkKeysLock = [[NSObject alloc] init];
 
-  // Compile SDK keys from configuration credentials
+  // Compile SDK keys from configuration credentials.
   for (GADMediationCredentials *credentials in configuration.credentials) {
-    NSString *sdkKey = credentials.settings[GADMAdapterAppLovinConstant.sdkKey];
-    if (sdkKey) {
-      [sdkKeys addObject:sdkKey];
+    NSString *sdkKey = credentials.settings[GADMAdapterAppLovinSDKKey];
+    if ([GADMAdapterAppLovinUtils isValidAppLovinSDKKey:sdkKey]) {
+      GADMAdapterAppLovinMutableSetAddObject(sdkKeys, sdkKey);
     }
   }
 
-  // Add SDK key from Info.plist if exists
-  if ([GADMAdapterAppLovinUtils infoDictionaryHasValidSDKKey]) {
+  // Add SDK key from Info.plist if it exists.
+  if ([GADMAdapterAppLovinUtils infoDictionarySDKKey]) {
     NSString *sdkKey = [GADMAdapterAppLovinUtils infoDictionarySDKKey];
-    [sdkKeys addObject:sdkKey];
+    if ([GADMAdapterAppLovinUtils isValidAppLovinSDKKey:sdkKey]) {
+      GADMAdapterAppLovinMutableSetAddObject(sdkKeys, sdkKey);
+    }
   }
 
-  // Log to the publisher that having more than 1 SDK key is... abnormal
-  if (sdkKeys.count > 1) {
-    [GADMAdapterAppLovinUtils
-        log:@"Found %lu SDK keys. Please remove any SDK keys you are not using from the AdMob UI.",
-            sdkKeys.count];
+  // Log to the publisher that having more than 1 SDK key is... abnormal.
```

Comment:
Should it be `sdkKeys.count != 1`? Or is only the error case when there are no SDK keys provided? If so, comment does not reflect the error case.

Reply:
We are only throwing an error only if the `sdkKeys` set is empty, so the comment is no longer correct here, removed.